source/core/token-reader.h

```diff
@@ -22,7 +22,7 @@ namespace Slang
         OpInc, OpDec, OpAddAssign, OpSubAssign, OpMulAssign, OpDivAssign, OpModAssign,
         OpShlAssign, OpShrAssign, OpOrAssign, OpAndAssign, OpXorAssign,
 
-        QuestionMark, Colon, RightArrow, At, Pound, PoundPound,
+        QuestionMark, Colon, RightArrow, At, Pound, PoundPound, Scope,
```

Comment:
I thought you'd planned to drop the changes to `source/core` since they end up not being required?

Reply:
I think somewhat confusingly the tokens as used by the compiler lexer uses this header - and that's why I had to add here. It's confusing though because (as we previously discussed) I removed support for lexing it from the TokenReader.cpp